documents/openapi/openapi.yaml

```
@@ -1012,6 +1012,31 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/EmailNotificationRequest'
+            examples:
+              email_example:
+                value:
+                  reference: string
+                  template_id: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+                  personalisation:
+                    full_name: John Smith
+                    claim_id: '123456'
+                  scheduled_for: string
+                  billing_code: string
+                  email_reply_to_id: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+                  email: "test@notification.com"
+              recipient_identifier_example:
+                value:
+                  reference: string
+                  template_id: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+                  personalisation:
+                    full_name: John Smith
+                    claim_id: '123456'
+                  scheduled_for: string
+                  billing_code: string
+                  email_reply_to_id: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+                  recipient_identifier:
+                    id_type: ICN
+                    id_value: 1008788712V759106
```

```suggestion
                    id_value: 1000000000V000000
```
I'm not sure where you got the ICN from but we don't want someone to get test notifications because they happen to be in the documentation.

---

I found that `id_value` in the same document as an example of `recipient_identifier` so I used that.